packages/terra-show-hide/src/_ShowHideButton.jsx

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './_ShowHideButton.module.scss';
+
+const cx = classNames.bind(styles);
+
+const KEYCODES = {
+  ENTER: 13,
+  SPACE: 32,
+  TAB: 9,
+};
+
+const ButtonTypes = {
+  BUTTON: 'button',
```

Comment:
Since we are on the other button types, do we need submit and reset? Those are used for forms, and I don't envision this being apart of a form.

Reply:
I don't believe the other variants will be used. I removed the unused variants: https://github.com/cerner/terra-core/pull/1956/commits/db61dc7961a60dfa0ef6406a735ae6c05d89d524